src/web/components/Menu/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import LinkText from '../LinkText';
+import List from '../List';
+import Item from '../Item';
+
+const renderItem = (item) => {
```

Comment:
You can destructure on the `(item)` part 😊

Reply:
Oki i will do that :+1: